docs/manuals/source/Appendix/ReleaseNotes.rst

```diff
@@ -24,13 +24,197 @@ This chapter concentrates on things to do when updating an existing Bareos insta
 
 .. _bareos-current-releasenotes:
 
-.. _bareos-1826-releasenotes:
+.. _bareos-1924-releasenotes:
 
-.. _bareos-18.2.6:
+.. _bareos-19.2.4:
+
+Bareos-19.2
+-----------
+
+Bareos-19.2.4
+~~~~~~~~~~~~~
+
+General Information
+^^^^^^^^^^^^^^^^^^^
+
+.. list-table:: Bareos 19.2.4 Release Information
+   :header-rows: 0
+   :widths: auto
+
+   * - **Release Date**
+     - 1 December 2019
+   * - **Database Version**
+     -  2171
+   * - **URL**
+     - https://download.bareos.com/bareos/release/19.2/
```

Comment:
Will also be published on https://download.bareos.org/bareos/release/19.2/

Reply:
If https://github.com/bareos/bareos/pull/314 get merged, I've to something more to the 19.2 release notes.

Reply:
Added another line pointing to bareos.org